charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -20,3 +20,163 @@ spec:
   - name: v1alpha1
     served: true
     storage: false
+  validation:
+    openAPIV3Schema:
```

Comment:
Due to the `status` subresource, I can't define `openAPIV3Schema` as `type: object` like the other CRD updates.

Reply:
I don't believe that this violates any structuring rules, as the root (`.spec`) isn't missing `type: object`, which would violate the second rule.

Reply:
This was only a problem in a 3.x cluster and has been resolved by upgrading to a 4.x cluster.